src/ScreenEdit.cpp

```diff
@@ -1526,6 +1528,21 @@ void ScreenEdit::Init()
 		SetDirty(true);
 	}
 
+
+	// There are two conditions for centering the notefield.
+	// 1. We're in EditMode
+	// 2. We're in PracticeMode, the preference is enabled,
+	//      multiplayer is not enabled, and the theme allows it.
+	ThemeMetric<bool> allow_center;
```

Comment:
Is this a new theme metric, or an existing one?

Reply:
Existing. `ScreenGameplay` actually loads it here:

https://github.com/itgmania/itgmania/blob/5d4f9dcb07493ed4c51d6be23e9b41978162305a/src/ScreenGameplay.h#L213
https://github.com/itgmania/itgmania/blob/5d4f9dcb07493ed4c51d6be23e9b41978162305a/src/ScreenGameplay.cpp#L356